### DISCUSSION TITLE
Migration Guid from electron to Tauri

### DISCUSSION THREAD

**Discussion by cawa-93** [tauri-apps/tauri#1869] [Ideas]
Today I was looking through the tauri's documentation and thinking about what I was missing in it.

I have never worked with Rust. And the "Usage" section is a bit confusing to me.

Given this, I think it would be easier to perceive the information if there was a guide that looked at some electron concepts and how they work in Tauri.

There could be something like this:
> The electronic entry point to your application is the `main.js` file. In this file you call `new BrowserWindow({ ... })` to create a new window with the specified parameters.
> In the Tauri it works like this: ` ... ` . And you can adjust the window settings like that: ` ... `. More -> link to a section of the documentation.

Or something like this:
> In the electron, integration with node modules is used to access operating system functions, such as file system, etc: `import fs from 'fs'`
> In the Tauri, access to OS functions is provided as follows: ` ... `. More -> link to a section of the documentation.

Over time, this guide could be supplemented by an increasing number of common user cases.

**Comment by SgtPooki** [comment]
Any update on this? Are there any examples of applications that have migrated from Electron to Tauri? I was able to find https://github.com/mul14/electron-to-tauri
